### PR TITLE
Run Ag Notebook in Build

### LINKED ISSUE #20: Earth Engine Map Layers Expire
## Bug Report

Tile Layers from raster results produced by Earth Engine disappear from interactive maps after EE token expires. Same issue as https://github.com/worldbank/DECAT_Space2Stats/issues/3

### Possible Fix

Use a [service account](https://developers.google.com/earth-engine/guides/service_account) to a authenticate EE and re-run notebook time the book is built. Maps should last a little longer (1 week) although it's not a permanent fix. Could set up an action to rebuild with more frequency?

Or, build and host tile layers differently...

### PR DESCRIPTION
**Pull request by andresfchamorro**
Small changes to execute notebook:
- [x] Authenticate Earth Engine via service account
- [x] Edit notebook metadata ([file level](https://myst-nb.readthedocs.io/en/v0.15.0/configuration.html#file-level-configuration)) to force run
- [x] Add notebook dependencies to project.toml

@g4brielvs Can you check if it makes sense and if I missed anything?

Closes #20

### CODE REVIEW

File: reports/agriculture/report.ipynb
Comment (g4brielvs) [inline reports/agriculture/report.ipynb:1]: @andresfchamorro If running this notebook on schedule, we will have to make the data available. For example, `../data/adm1_crop_count.csv`. Do you think we this should be versioned in the repository?

**Review by g4brielvs — COMMENTED** [review]
@andresfchamorro Thanks for opening this PR! I've made [a few changes](https://github.com/datapartnership/niger-economic-monitoring/pull/21/files), namely moved the report to `reports/`, updated the config and added the report to the table of contents. What do you think?

File: reports/agriculture/report.ipynb
Comment (andresfchamorro) [reply reports/agriculture/report.ipynb:1]: I don't foresee that table to change, that's just the result of zonal stats for the reference crop map. In other cases like red sea, absolutely, although output data is typically written to SharePoint

Comment (g4brielvs) [reply reports/agriculture/report.ipynb:1]: @andresfchamorro In that case, do you think the table can be committed to the repository I usually would discourage versioning data on the repository, but if the notebook is to be executed on schedule, the data must be available somewhere.

Comment (andresfchamorro) [reply reports/agriculture/report.ipynb:1]: oh yes, i didn't realize it wasn't tracked. let me add it

**Review by g4brielvs — APPROVED** [review]
@andresfchamorro Let's see if it will work!

**Pull request merged**